pkg/cluster/cluster_config.go

```diff
@@ -135,20 +136,21 @@ type Release struct {
 }
 
 func GetRelease(cli client.Client) (Release, error) {
-	initRelease := Release{
+	buildVer, err := semver.Parse(odhversion.Version)
+	if err != nil {
 		// dummy version set to name "", version 0.0.0
+		buildVer = semver.Version{}
+	}
+	initRelease := Release{
 		Version: version.OperatorVersion{
-			Version: semver.Version{},
+			Version: buildVer,
 		},
 	}
-	// Set platform
-	platform, err := GetPlatform(cli)
-	if err != nil {
-		return initRelease, err
-	}
+	// Set platform. error can be ignored, it's "" in that case
+	platform, _ := GetPlatform(cli)
 	initRelease.Name = platform
 
-	// For unit-tests
+	// For unit-tests. TODO: Is it needed?
```

Comment:
Not anymore after https://github.com/opendatahub-io/opendatahub-operator/pull/1028/files#diff-3bd42200a1af1b71af4626a7e6092cbaccf8a382fef07fa34119595c7b09f2f4R139